SPECS/intel-ipsec-mb/intel-ipsec-mb.spec

```
@@ -0,0 +1,150 @@
+%global githubver    %{major}.%{minor}
+%global githubfull   %{githubname}-%{githubver}
+# disable producing debuginfo for this package
+%global debug_package %{nil}
+Summary:        IPSEC cryptography library optimized for Intel Architecture
+Name:           %{githubname}
+Version:        %{fullversion}
+Release:        2%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://github.com/intel/%{githubname}
+Source0:        https://github.com/intel/%{githubname}/archive/v%{githubver}.tar.gz#/%{githubfull}.tar.gz
+# Copyright (c) 2017-2022, Intel Corporation
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+#     * Redistributions of source code must retain the above copyright notice,
+#       this list of conditions and the following disclaimer.
+#     * Redistributions in binary form must reproduce the above copyright
+#       notice, this list of conditions and the following disclaimer in the
+#       documentation and/or other materials provided with the distribution.
+#     * Neither the name of Intel Corporation nor the names of its contributors
+#       may be used to endorse or promote products derived from this software
+#       without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# Versions numbers
+BuildRequires:  gcc >= 4.8.3
+BuildRequires:  make
+BuildRequires:  nasm >= 2.14
+ExclusiveArch:  x86_64
+
+%description
+IPSEC cryptography library optimized for Intel Architecture
+
+%package -n intel-ipsec-mb-devel
+Summary:        IPSEC cryptography library optimized for Intel Architecture
+Group:          Development/Tools
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+ExclusiveArch:  x86_64
+
+%description -n intel-ipsec-mb-devel
+IPSEC cryptography library optimized for Intel Architecture
+
+For additional information please refer to:
+https://github.com/intel/%{githubname}
+
+%prep
+%autosetup -n %{githubfull}
+
+%if 0%{?rhel} && 0%{?rhel} < 8
+%ldconfig_post
+
+%ldconfig_postun
+%endif
+
+%build
+cd lib
+make EXTRA_CFLAGS='%{optflags}' %{?_smp_mflags}
```

Nit: The `%make_build` macro inserts the `%{?_smp_mflags}` flags and enables more verbose logging, which can be useful if the package build breaks for some reason.

```suggestion
%make_build EXTRA_CFLAGS='%{optflags}'
```